### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by abulte**
Nothing about the business code, all about the tests :-/

### CODE REVIEW

**Comment by abulte** [comment]
@davidbgk should we start a `dev` branch with this? ATM I'm not confident in merging this in `master` if `master` is to be shippable in production at all times.

**Comment by davidbgk** [comment]
@abulte Good question. I would say let's merge that one and deploy to verify that dependencies update doesn't break anything. Then iterate with feature branches that might be merged directly on master? Or do we want a dev env to make some live tests prior to push to prod?

File: tests/test_integrations.py [hunk 1]
```diff
@@ -62,8 +40,12 @@ def test_cache_report(runner_factory, web_session):
     storage.expire_cache.assert_called_once_with(default_key, cache_duration)
 
 
-def test_retrieve_url_with_group(runner_factory, web_session):
-    runner = runner_factory(HttpService)
+def test_retrieve_url_with_group(
+    runner_factory,
+    web_session,
+    web_container_config
+):
```
Comment (davidbgk) [inline tests/test_integrations.py]: I'm not used to that kind of indentation, proposition:

```python
def test_retrieve_url_with_group(
        runner_factory, web_session, web_container_config):
    runner = runner_factory(web_container_config, HttpService)
```

**Comment by abulte** [comment]
@davidbgk we'll need a dev env pretty soon if we want to test the web hook logic and the udata integration against the dev version of udata, don't you think?

**Comment by abulte** [comment]
Logging is broken, I'm looking into it.

**Comment by abulte** [comment]
Logbook got a lot less fun since 0.10.0 - you now have to configure a default handler, more boilerplate... Since what we have works and this should not be a critical component, I reverted to Logbook 0.9.1.

**Comment by davidbgk** [comment]
> Since what we have works and this should not be a critical component, I reverted to Logbook 0.9.1.

Alright, we may want to remove Logbook completely and switch to logging if it's too cumbersome.

> we'll need a dev env pretty soon if we want to test the web hook logic and the udata integration against the dev version of udata, don't you think?

Sure, @odtvince can you provide us that?

**Comment by abulte** [comment]
@davidbgk I got the latest version of Logbook to work w/ minimal additional boilerplate. I also created a dev branch from master to merge this. If it's all good for you I'll merge.

**Comment by davidbgk** [comment]
Looks good to me, let's get this merged! 👍

**Pull request merged**